README.md

```diff
@@ -44,7 +44,7 @@ Ensure that you deploy the `/static` directory, as Prebid Server requires those
 
 ## Developing
 
-Prebid Server requires [Go](https://go.dev) version 1.21 or newer. You can develop on any operating system that Go supports; however, please note that our helper scripts are written in bash.
+Prebid Server requires [Go](https://go.dev) version 1.22 or newer. You can develop on any operating system that Go supports; however, please note that our helper scripts are written in bash.
```

Comment:
Gah my mistake. It appears the README does not need to be updated. 1.21 is actually correct here as you can see that is what we declare in `go.mod`. We are currently supporting 1.21 and 1.22. The dev container and our github actions though should all run with 1.22. Can you undo this change?

Reply:
Hi @bsardo 

Sure.

See "Update README.md to put back Go 1.21" -> https://github.com/prebid/prebid-server/pull/3983/commits/caf0c5694501c7c3799db4eff33f6da5ca5212d6